### DISCUSSION TITLE
Brew-foo

### DISCUSSION THREAD

**Discussion by Rockyspade** [community/community#145074] [Repositories]
### Select Topic Area

Question

### Body

Brew foo is out date my keyboard made an error somehow switched to revert something I didn't the get change switch back because I sign to Linux not employee I don't trust Microsoft guess down try set myLabtop to I made my to #8542 so have Files mistake not filed my brew-foo outdated get back repo

**Comment by Rockyspade** [comment]
Try revert brew-foo #2 #4  production

**Reply by Rockyspade** [reply]
That's it yep [Brew-foo]

**Reply by Rockyspade** [reply]
> Try revert brew-foo #2 #4  production 

Update

**Reply by Rockyspade** [reply]
👍👍

**Reply by Rockyspade** [reply]
[Brew-foo] status fetch init rebase and smashed to merge checkout version

**Reply by Rockyspade** [reply]
My [Linux-Windows10.32x86]  [diestros]  bug issues with malfunction  GitHub@desktop open won't Install [ brew.sh] @Rockyspade @queenofcorgis

**Comment by queenofcorgis** [comment]
Hey there @Rockyspade , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction. Would you mind elaborating more on brew foo?

**Reply by Rockyspade** [reply]
Hello Queen of Corgis it's about repository Greenbrew it's not my list repo! because I was working all the sudden in my PC my target popover is working one part is working but the button is not in development environment and event.@GitHub desktop Windows(x86)

**Reply by Rockyspade** [reply]
[topic branch] I labeled [Greenbrew] from  [ Homebrew] [foo-brew]  myaccountsync bug issues automation [branch] [ logrus-trace-level-logging] plugin **test** @Rockyspade  update status Sync 
Checkout
